### ISSUE TITLE
Mismatch of label structure between example_data/stg3 and custom_dataset class

### ISSUE THREAD

**Issue by anantterkar** [raulkviana/MSE-CNN-Implementations#4]
Hi Raulkviana,

I have been trying to reproduce the results cited in QTMT paper. As there is no mention of training and inference scripts, I have written my own scripts to run the inference. I am facing an issue where the data given in example_data/stg3 doesn't match with the expected entry structure for CUDatasetStg3Compl. 
This is my sample from the script corresponding to the above mentioned issue:

`            entry = dataset.select_entry(i)
            sample = dataset.get_sample(entry)
`

This failes at second line due to the following reason:
>> orig_pos_x = entry[11][1]

entry from the dataset.select_entry(i) has following structure:
`
elif stg_name == "Stage 3":
                # Current dataset structure has RD_costs first, CU_Y last
                rd_costs = sample[0]
                cu_pos_stg2 = torch.tensor(sample[1])
                cu_pos = torch.tensor(sample[2])
                cu_size_stg2 = torch.tensor(sample[3])
                cu_size = torch.tensor(sample[4])
                split_stg2 = int(sample[5])
                true_label = int(sample[6])
                cu = torch.tensor(sample[7]).unsqueeze(0).float()  # move CU_Y to first channel

                ap = 64
                sizes = cu_size.unsqueeze(0)
                coords = cu_pos.unsqueeze(0)
                splits = torch.tensor([split_stg2])

                pred, _, _ = model_stage(cu.to(device), ap, splits, sizes, coords)
`
Please suggest a fix for this as I do not wish to modify the dataset classes. Or if I am loading it an incorrect manner, please help me out.
Thanking you
Regards
